skimage/filters/_gaussian.py

```diff
@@ -55,7 +55,7 @@ def gaussian(image, sigma=1, output=None, mode='nearest', cval=0,
     -----
     This function is a wrapper around :func:`scipy.ndi.gaussian_filter`.
 
-    Integer arrays are converted to float.
+    Integer arrays are converted to float, if output specified as float.
```

Comment:
I personally, would not allow this case, since the documentation clearly stipulates that `output` is an array, but well...